lib/elixir/src/elixir_code_server.erl

```diff
@@ -26,6 +27,7 @@ start_link() ->
 init(ok) ->
   %% The table where we store module definitions
   _ = ets:new(elixir_modules, [set, protected, named_table, {read_concurrency, true}]),
+  _ = ets:new(elixir_opened_modules, [set, protected, named_table, {read_concurrency, true}]),
```

Comment:
We should not use a separate table  for this. When we store the module in the `elixir_modules` table, we should have a field in there that says if it is `open` / `read_only`.